unittests/PhasarLLVM/DataFlow/Mono/InterMonoTaintAnalysisTest.cpp

```diff
@@ -391,4 +391,4 @@ TEST(InterMonoTaintAnalysisTestNF, TaintTest_05) {
 int main(int Argc, char **Argv) {
   ::testing::InitGoogleTest(&Argc, Argv);
   return RUN_ALL_TESTS();
-}
+}
```

Comment:
Please revert -- we always want to have a trailing line feed at the end of a file